modular-docs-manual/content/topics/using_text_snippets_or_text_fragments.adoc

```diff
@@ -1,26 +1,19 @@
-// Base the file name and the ID on the module title. For example:
-// * file name: my-concept-module-a.adoc
-// * ID: [id="my-concept-module-a_{context}"]
-// * Title: = My concept module A
-
-// The ID is an anchor that links to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
 [id="using_text_snippets_or_text_fragments_{context}"]
-// The `context` attribute enables module reuse. Every module ID includes a variable that sets the context, such as {context}, which ensures that the module has a unique ID even if it is reused multiple times in a guide.
-= Text Snippets or Text Fragments (Pseudo-modules)
-//In the title of concept modules, include nouns or noun phrases that are used in the body text. This helps readers and search engines find the information quickly.
-//Do not start the title of concept modules with a verb. See also _Wording of headings_ in _The IBM Style Guide_.
-
-[NOTE]
-The following standard is recommended when the documentation is being maintained without a Content Management System (CMS) capable of managing complex interrelations between modules.
-
-The use of reusable text snippet files (or text fragment files) is discouraged due to the complications that can arise due to the complexity they introduce.
 
-Snippet (fragment) file use should be limited to:
+= Text Snippets or Text Fragments
 
-* Standardized admonitions (such as 'Technology preview' and 'Beta' text).
-* Where there is an existing standard between the upstream and downstream communities.
+A text snippet or text fragment is a small section of text that is stored in an Asciidoc file. Text snippets contain content that is resused in multiple modules or assemblies, for example:
 
+* A step or series of steps in a procedure
+* A disclaimer, for example for technology preview or beta releases
 
-//.Additional resources
+NOTE: A text snippet is not a module. The Asciidoc file that contains the text snippet cannot contain any other information. A module cannot contain an include statement for another module.
 
-//* A bulleted list of links to other material closely related to the contents of the concept module.
+.Procedure
+. Create the text snippet Asciidoc file and store it in your snippets folder.
```

Comment:
I'll lick a fingertip and then reach out for the third rail...vanilla Asciidoctor is perfectly happy with a snippets folder.  Does the tooling that we're not supposed to be overly focused on also support a snippets folder?

Other than mental harm, is it problematic to put the text frags in a modules folder?--I get it that we just stated it's not a module, then we're possibly advising to co-locate with the modules.

Reply:
Yes, the Publishing-Platform-That-Shall-Not-Be-Named needs to have fragments in a single location. This location is added to the YAML file.

Reply:
I'd swear that the file now includes mention of a "fragments" folder, but I did not checkout the PR.  I share that in case it was an overzealous change from "snippets" to "fragments." I'm good with whatever you propose.